### ISSUE TITLE
Hate Speech Detection Model Development

### ISSUE THREAD

**Issue by lovek28** [TanviTakkar/Hate-Speech-Detection#44]
Fine-tune a pre-trained BERT model on the hate speech dataset.
Evaluate the model's performance using metrics like accuracy, precision, recall, and F1-score.

**Comment by aniketgupta97** [comment]
Hate speech with topic model trained and tested
